compute_tools/src/rsyslog.rs

```diff
@@ -75,3 +79,69 @@ pub fn configure_audit_rsyslog(
 
     Ok(())
 }
+
+#[instrument(skip_all)]
+fn pgaudit_gc_main_loop(log_directory: String) {
```

Comment:
I guess you did some experiments, but reimplementing log rotation is ugh. Any chance we can reuse `logrotate` or something like that?

Reply:
I suggest moving the logrotate cronjob to cron.hourly (that's what I did before, at least) some time in the future